protocol/binary/reader.go

```diff
@@ -202,6 +207,19 @@ func (br *Reader) read(bs []byte, off int64) (int64, error) {
 	return off, err
 }
 
+// copyN copies the given number of bytes starting at the given position into
+// the Writer.
+func (br *Reader) copyN(w io.Writer, off int64, n int64) (int64, error) {
```

Comment:
Forgot to push the change into this PR. Fix: https://github.com/thriftrw/thriftrw-go/pull/165